read.go

```diff
@@ -226,24 +230,22 @@ func readNativeFrames(d dicomio.Reader, parsedData *Dataset, fc chan<- *frame.Fr
 		buf := make([]int, int(pixelsPerFrame)*samplesPerPixel)
 		for pixel := 0; pixel < int(pixelsPerFrame); pixel++ {
 			for value := 0; value < samplesPerPixel; value++ {
-				if bitsAllocated == 8 {
-					val, err := d.ReadUInt8()
-					if err != nil {
-						return nil, bytesRead, err
+
```

Comment:
nit: no need for a new line here (but I can also just fix in a cleanup pr later, don't need to block this).

```suggestion
```